graphql/notification/mutations.ts

```diff
@@ -79,6 +79,14 @@ export class MutateNotificationResolver {
         return true;
     }
 
+    @Mutation((returns) => Boolean)
+    @Authorized(Role.ADMIN)
+    async notificationDelete(@Arg('notificationId') notificationId: number): Promise<boolean> {
```

Comment:
@Arg(..., type => GraphGLInt) as otherwise the type is inferred as Float, which is slightly wrong (but I made the same mistake in many other places already)